test/FunctionalTests/Microsoft.OData.Core.Tests/ScenarioTests/Roundtrip/JsonLight/SingletonBatchRoundtripJsonLightTests.cs

```diff
@@ -995,6 +997,30 @@ public void BatchJsonLightTestUsingJson()
             BatchJsonLightTestUsingBatchFormat(BatchFormat.ApplicationJson, 0);
         }
 
+#if NETCOREAPP3_1_OR_GREATER
+        [Fact]
+        public void BatchJsonLightTest_WithODataUtf8JsonWriter()

```

Comment:
```suggestion
        public void MultipartMixedBatchTest_WithODataUtf8JsonWriter()
```

Reply:
I was using the same name from exists tests. The convention in these test cases is that when the test case uses multipart, it's not mentioned in the name, and when it uses JSON, it uses the `UsingJson` suffix. I can change the existing tests to use `UsingMultipartMixed` suffix.